### PR TITLE
Fixed issue #6371

### PR DESCRIPTION
**Pull request by akathuria289**
## Issue Reference
#6371

## Description
Flash messages appear when catalog or cart rule applied

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Not working properly, The flash message will show only when the cart or catalog rule will be applied.

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul It`s working as you can see in the reference video - https://www.awesomescreenshot.com/video/10102502?key=c800170ac36a2cca41e4975edaa167e4

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
Please disable the cart rule and try again to place an order. 
If the cart rule is **disabled** then the flash message **should not be visible.**

**Comment by akathuria289** [comment]
@Sanjaybhattwebkul Fixed

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Comment by Sanjaybhattwebkul** [comment]
@akathuria289 
This is **working fine** please find the screenshot [Here](https://prnt.sc/Ra9N0FSEAG1x)

File: packages/Webkul/Shop/src/Resources/lang/en/app.php [hunk 1]
```diff
@@ -543,6 +543,7 @@
             'check-billing-address'     => 'Please check billing address.',
             'specify-shipping-method'   => 'Please specify shipping method.',
             'specify-payment-method'    => 'Please specify payment method.',
+            'rule-applied'              => 'Cart rule applied',
```
Comment (devansh-webkul) [inline packages/Webkul/Shop/src/Resources/lang/en/app.php:546]: Add remaining files key.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

File: packages/Webkul/Shop/src/Http/Controllers/OnepageController.php [hunk 2]
```diff
@@ -58,6 +58,10 @@ public function index()
         }
 
         $cart = Cart::getCart();
+        
+        if ($cart->applied_cart_rule_ids != "") {
```
Comment (jitendra-webkul) [inline packages/Webkul/Shop/src/Http/Controllers/OnepageController.php:62]: Use single quotes here

**Comment by akathuria289** [comment]
Done @devansh-webkul Sir @jitendra-webkul Sir

**Review by devansh-webkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**